scripts/checks.sh

```diff
@@ -179,6 +179,14 @@ function test_disk_expansion()
 	fi
 }
 
+function test_data_partition_mounted()
+{
+	DATA_MOUNTPOINT=$(lsblk -J -o NAME,MOUNTPOINT | jq -r '.blockdevices[].children[] | select (.mountpoint == "/mnt/data") | .name')
```

Comment:
I think we can just check if we get a result or not, since we're not actually checking what is mounted there:

```suggestion
	if lsblk -J -o NAME,MOUNTPOINT | jq -er '.blockdevices[].children[] | select (.mountpoint == "/mnt/data") | length == 1' > /dev/null; then
```

Reply:
That seems good to me.  I've had to adjust that in a couple minor ways:

- `.children[]?`, to avoid `cannot iterate over null` error
- `.length >= 1`, as the object passed to the `length` directive is ` {"name": "mmcblk0p6", "mountpoint": "/mnt/data"}` (length of 2)